corehq/form_processor/utils/sql.py

```diff
@@ -61,7 +61,7 @@ def form_adapter(form):
         form.deletion_id,
         form.server_modified_on,
     ]
-    return ObjectAdapter(fields, XFormInstance_DB_TABLE)
+    return ObjectAdapter(fields, XFormInstance.Meta.db_table)
```

Comment:
This should be `_meta` (even though `.Meta` works)

Reply:
2000a5575b0fa4debc200de5511520d625149d3c